packages/dds/tree/src/simple-tree/api/viewSchemaToSimpleSchema.ts

```diff
@@ -126,11 +126,17 @@ function fieldSchemaToSimpleSchema(schema: FieldSchema): SimpleFieldSchema {
 	}
 
 	const allowedTypes = allowedTypesFromFieldSchema(schema);
-	const result = {
+	const result: SimpleFieldSchema = {
 		kind: schema.kind,
 		allowedTypes,
 	};
 
+	// Don't include "description" property at all if it's not present.
+	if (schema.metadata?.description !== undefined) {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		(result as any).description = schema.metadata.description;
```

Comment:
```suggestion
		(result as Mutable<Pick<typeof result, "description">>).description = schema.metadata.description;
```
A little verbose, but technically the most precise way to express what you're trying to do and it gets rid of the `any`.

Reply:
Ah, there _is_ a mutable type! I tried that, and intellisense wasn't helpful.